curiosrc/seal/task_treed.go

```diff
@@ -63,6 +109,20 @@ func (t *TreesTask) Do(taskID harmonytask.TaskID, stillOwned func() bool) (done
 	}
 	sectorParams := sectorParamsArr[0]
 
+	sref := storiface.SectorRef{
+		ID: abi.SectorID{
+			Miner:  abi.ActorID(sectorParams.SpID),
+			Number: abi.SectorNumber(sectorParams.SectorNumber),
+		},
+		ProofType: sectorParams.RegSealProof,
+	}
+
+	// Fetch the Sector to local storage
+	fsPaths, pathIds, release, err := t.sc.PreFetch(ctx, sref, &taskID)
```

Comment:
I'd defer release here instead of in TreeD so that it is always called - between here and the TreeD call a lot can still fail, and we really don't want to leak storage reservations

Reply:
Moved the defer